plugins/by-name/wezterm/default.nix

```
@@ -0,0 +1,21 @@
+{ lib, ... }:
+let
+  inherit (lib.nixvim) defaultNullOpts;
+in
+lib.nixvim.neovim-plugin.mkNeovimPlugin {
+  name = "wezterm";
+  originalName = "wezterm.nvim";
+  package = "wezterm-nvim";
+
+  maintainers = [ lib.maintainers.samos667 ];
+
+  settingsOptions = {
+    create_commands = defaultNullOpts.mkBool true ''
+      Whether to create plugin commands. Plugin default is true.
```

```suggestion
      Whether to create plugin commands.
```
We don't need to specify the default in the description, the `true` from the arguments will generate that in the documentation.